lib/inferno/exceptions.rb

```diff
@@ -39,6 +39,16 @@ def result
       end
     end
 
+    class ErrorInValidatorException < TestResultException
+      # This extends TestResultException instead of RuntimeError
```

Comment:
This is documentation for the class rather than the method, so it should go above the class declaration. If you run `yardoc` and look at the generated docs, you'll see this is treated as method documentation.

Reply:
Ugh. I didn't realize this would show up in external doco at all, it was meant to be just a quick internal implementation note. I'll clean that up

Reply:
Adding a newline between the comment and the method definition might fix that.

Reply:
Unfortunately it needs two newlines which rubocop doesn't like. I moved the comment to above the class declaration and added a few words to make a it a more complete class description. Probably also could have just deleted it but this class hierarchy feels like the kind of thing that someone might see later and wonder why.

Reply:
Yeah, I thought this was a worthwhile comment.